interfaces/identity.js

```diff
@@ -0,0 +1,22 @@
+type chrome$AccountInfo = string;
+
+type chrome$identity = {
+  getAccounts(callback: (accounts: Array<chrome$AccountInfo>) => void): void,
+  getAuthToken(details: {
+    interactive?: boolean,
+    account?: chrome$AccountInfo,
+    scopes?: Array<string>
+  }, callback?: (token: ?string) => void): void,
+  getProfileUserInfo(callback: (userInfo: { email: ?string, id: ?string }) => void): void,
```

Comment:
I think `email` and `id` should just be typed as `string`. They'll just be empty strings if the permissions are not specified according to https://developer.chrome.com/apps/identity#method-getProfileUserInfo